tests/helpers/datasets.py

```diff
@@ -19,7 +19,6 @@
 from typing import Optional, Sequence, Tuple
 
 import torch
-from torch import Tensor
```

Comment:
why is this wrong?

Reply:
Just making sure both implementations match exactly